scripts/release/bump-package-version.js

```diff
@@ -19,7 +19,10 @@ if (!semver.valid(newVersion)) {
   );
 }
 
-if (semver.gte(currentVersion, newVersion)) {
+if (
+  semver.gte(currentVersion, newVersion) &&
+  semver.diff(newVersion, currentVersion) !== 'prerelease'
```

Comment:
Why do we need this?

Reply:
We need this change because in semver `4.X.X-experimental-typescript.X < 4.X.X` and `semver.gte` will then return false. This is a problem if we want to do the TypeScript release after the stable one (what we'll most likely do).

With this change I allow us to publish prereleases of the `currentVersion`.

Reply:
I see – I think that'd be worth adding a comment explaining why this line is here then.